src/config.ts

```diff
@@ -350,21 +362,17 @@ function loadPlugins(
     const pluginLoc = require.resolve(name, {paths: [process.cwd()]});
     const plugin = require(pluginLoc)(config, options);
     plugin.name = plugin.name || name;
-    if (plugin.defaultBuildScript && !plugin.input) {
+    if (plugin.build) {
```

Comment:
We should also handle `transform()` here, too, because that’s a legacy thing.

Thoughts: we check for `name` and/or `resolve.*` to determine first whether a plugin is v1 or v2. If v1, we add those fields and change `build -> load`, `transform -> ???`. If v2 we load as-is.

We can still normalize altogether afterward if needed

Reply:
talked in person: no longer a legacy thing! transform() is supported going forward, in this model